cloudconfig/podcfg/image.go

```diff
@@ -15,6 +15,7 @@ import (
 const (
 	JujudOCINamespace = "jujusolutions"
 	JujudOCIName      = "jujud-operator"
+	JujuK8sAgentName  = "k8sagent"
```

Comment:
have we already split `k8sagent` from `jujud-operator`(the build process)?

Reply:
Yep that was landed earlier